extensions/amp-story/1.0/media-performance-metrics-service.js

```diff
@@ -141,8 +142,9 @@ export class MediaPerformanceMetricsService {
    * method has to be called right before trying to play the media. This allows
    * to reliably record joint latency (time to play), as well initial buffering.
    * @param {!HTMLMediaElement} media
+   * @param {boolean=} isActivePage
    */
-  startMeasuring(media) {
+  startMeasuring(media, isActivePage = false) {
```

Comment:
This method signature looks wrong (why do you need to send such a param to start tracking a media?), and it's not true: we're not trying to determine if this is the active page, but we want to know if it's the first page.
It'll always be the active page.